### PR TITLE
Add Working with Polaris page

### PR DESCRIPTION
**Pull request by selenehinkley**
This adds a page to Polaris.Shopify.com in the Contributing section to cover points from a Service Level Agreement but with a more approachable and less legalese angle.

### 🎩
Screen Shot 2022-08-05 at 7 59 53 AM

### CODE REVIEW

File: polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  - open source
+  - contribution guide
+---
+
+Through supporting teams, we’ve learned what works well, and not so well. We’ve outlined these things so that you can create a successful plan. This guide goes into detail, but here are the highlights.
+
+### TL;DR
+
+- High pressure deadlines are stressful, but the Polaris team can’t be pulled in to unblock. Think about the system early in the design process and plan for any system updates or additions ahead of the build stage. The Polaris team is here to support you while planning. Start a [GitHub discussion](https://github.com/Shopify/polaris/discussions/6750) for a larger contribution. For smaller contributions like bug fixes, read the [contribution guide](/contributing).
+- For the system to stay healthy and scale, contributions require a certain level of [quality](/contributing/quality-contributions)
```
Comment (jjgali) [inline polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md]: Is this a placeholder link for the quality principles that @raquelbreternitz and I are working on? Just want to note if so, that way we can update it when our PR goes up.

**Review by jjgali — APPROVED** [review]

File: polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md (hunk 1 above)
Comment (selenehinkley) [reply polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md]: Yep! It's a placeholder :) thanks for checking

File: polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+| Feature request                                                                  | For now, feature requests will be closed and reviewed against our existing backlog and roadmap. We'll evaluate interest in the request and prioritize it for development if there is enough interest and alignment with our product roadmap. | If you're interested in the feature please upvote the request in [GitHub](https://github.com/Shopify/polaris/discussions/6750) to let the team know.                                                                                                     |
+| Pull request                                                                     | Will review [PRs](https://github.com/Shopify/polaris/pulls) for system alignment, user need, and contribution quality.                                                                                                                       | Leave comments for code review.<br/><br/> Upvote the request in [GitHub](https://github.com/Shopify/polaris/discussions/6750) to express interest and let the team know you want this feature.                                                           |
```
Comment (yurm04) [inline polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md:100]: seems like a few of these link to the discussions page even when not relevant to discussions. is this intentional?

File: polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+| -------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Bug report                                                                       | Prioritized against our backlog and roadmap. If there's a clear or urgent need for a fix we'll add it to our backlog.                                                                                                                        | Leave comments with additional information on how to reproduce the bug and how it's affecting your use case.<br/><br/> If you're interested in a fix, [upvote](https://github.com/Shopify/polaris/discussions/6750) the bug report to let the team know. |
```
Comment (chloerice) [inline polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md]:
```suggestion
| Bug report                                                                       | Prioritized against our backlog and roadmap. If there's a clear or urgent need for a fix we'll add it to our backlog.                                                                                                                        | Leave comments with additional information on how to reproduce the bug and how it's affecting your use case.<br/><br/> If you're interested in a fix, [upvote](https://github.com/Shopify/polaris/issues/new?assignees=&labels=%F0%9F%90%9BBug&template=ISSUE.md) the bug report to let the team know. |
```

File: polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+| Feature request                                                                  | For now, feature requests will be closed and reviewed against our existing backlog and roadmap. We'll evaluate interest in the request and prioritize it for development if there is enough interest and alignment with our product roadmap. | If you're interested in the feature please upvote the request in [GitHub](https://github.com/Shopify/polaris/discussions/6750) to let the team know.                                                                                                     |
```
Comment (chloerice) [inline polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md]:
```suggestion
| Feature request                                                                  | For now, feature requests will be closed and reviewed against our existing backlog and roadmap. We'll evaluate interest in the request and prioritize it for development if there is enough interest and alignment with our product roadmap. | If you're interested in the feature please upvote the request in [GitHub](https://github.com/Shopify/polaris/issues/new?assignees=&labels=Feature+request&template=FEATURE_REQUEST.md) to let the team know.                                                                                                     |
```

**Review by chloerice — APPROVED** [review]
This looks great, Selene!! Added a couple of link changes that Yurms found 🔗 and then I think it's good to 🚢

File: polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md (hunk 2 above)
Comment (selenehinkley) [reply polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md:100]: apologies! I thought discussions were the only place to upvote so I threw the links in there. thanks @chloerice for updating :D

File: polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+-  #polaris in Slack if you work at Shopify and have a general inquiry
+
+As you’re building you may need custom styles. In those cases, always use Polaris [tokens](/tokens).
+
+## Product team tips
+With some early planning and intentional systems thinking, the build stage will go a lot smoother.
+
+### Plan for systems changes ahead of time
+
+The Polaris team should never be pulled in to unblock a team from shipping something quickly—we’re here to support and guide teams when they’re planning their strategic system changes ahead of time. Teams at Shopify should start working on their system needs early in the design and build process and are expected to make [quality system contributions](link to new quality guidelines) ahead of their ship dates.
```
Comment (yurm04) [inline polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md:29]:
```suggestion
The Polaris team should never be pulled in to unblock a team from shipping something quickly—we’re here to support and guide teams when they’re planning their strategic system changes ahead of time. Teams at Shopify should start working on their system needs early in the design and build process and are expected to make [quality system contributions](/contributing#what-makes-a-good-contribution) ahead of their ship dates. 
```

Comment (yurm04) [reply polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md:29]: swapping for the current quality guidelines that @raquelbreternitz and @jjgali shipped, might make sense to swap them out later when we have more quality docs

Comment (selenehinkley) [reply polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md:29]: fixed with https://github.com/Shopify/polaris/pull/6862

**Pull request merged**